### ISSUE TITLE
Experience Orb spawner does not add xp to xp players bar

### ISSUE THREAD

**Issue by cloakfox** [dnyferguson/MineableSpawners#59]
Using version 3.0.11 on paper 1.16.4

The orbs that get spawned from the spawner do not add onto your experience bar

**Comment by cloakfox** [comment]
Side note: I've disabled spawner AI in spigot.yml

**Comment by dnyferguson** [comment]
Are you using actual exp spawners? and have you tried with AI on just to see?

**Comment by cloakfox** [comment]
> Are you using actual exp spawners? and have you tried with AI on just to see?

Yes I'm using actual exp spawners, that spawn xp orbs, I have not tried with AI on but I doubt that would fix it, I can't run that either on production because of the performance hit

**Comment by cloakfox** [comment]
this is still an issue
